scripts/i2c_lcd_driver.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please make sure that the code works with Python 3.5 and add a Python3 shebang at the beginning of the file.

Reply:
The comment is reflected in and resolved by the latest commit.